pandas/core/series.py

```diff
@@ -2266,36 +2266,61 @@ def _binop(self, other, func, level=None, fill_value=None):
 
     def combine(self, other, func, fill_value=None):
         """
+        Combine the Series with a Series or Scalar according to `func`.
+
         Perform elementwise binary operation on two Series using given function
-        with optional fill value when an index is missing from one Series or
-        the other
+        with optional `fill_value` when an index is missing from the Series or
+        the other value.
```

Comment:
After reading the whole summary I still don't understand what this method does. Can you try to explain it more clearly (I know the description comes from the original docstring and not your changes).